### PR TITLE
fix: Correct handling of WebGL compatibility errors

### LINKED ISSUE #720: "WebGL2 not available" error is causing funnel drops

### PR DESCRIPTION
**Pull request by sandrade-dcl**
# What does this PR fix? 
## Fixes https://github.com/decentraland/unity-renderer/pull/720
We should check the WebGL compatibility of the current browser and show our "not supported" error screen if it is not supported:

image

## How to test it?
1. Open Firefox and type `about:config` in the url bar.
2. Search the flag `webgl.disabled` and turn it to `false`.
3. Go to https://play.decentraland.zone/branch/fix/Handle-the-WebGL-support/index.html?ENV=org
4. Notice that the "not supported" error screen is showed.
5. Go to the `webgl.disabled` again and turn it to `true` now.
6. Go to https://play.decentraland.zone/branch/fix/Handle-the-WebGL-support/index.html?ENV=org again
7. Notice that the world is loaded correctly.

### CODE REVIEW

**Review by kuruk-mm — APPROVED** [review]
Great!

**Review by pravusjif — APPROVED** [review]
Working as expected great job!

File: kernel/packages/shared/comms/browser.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export function isWebGLCompatible() {
+  // Create canvas element. The canvas is not added to the document itself, so it is never displayed in the browser window.
+  var canvas = <HTMLCanvasElement>document.createElement("canvas");
+  var gl = canvas.getContext("webgl") || canvas.getContext("experimental-webgl") || canvas.getContext("webkit-3d") || canvas.getContext("moz-webgl");
```
Comment (menduz) [inline kernel/packages/shared/comms/browser.ts]:
```suggestion
  var gl = canvas.getContext("webgl2");
```

I would check directly against webgl2 without fallbacks, to ensure maximum compatibility

**Review by menduz — CHANGES_REQUESTED** [review]

File: kernel/packages/shared/comms/browser.ts (hunk 1 above)
Comment (sandrade-dcl) [reply kernel/packages/shared/comms/browser.ts]: Done in https://github.com/decentraland/explorer/pull/2472/commits/4be1aac39c3c677cbb336542a8ccf422829d64ec. Thanks!!

**Review by menduz — APPROVED** [review]

**Pull request merged**